docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -66,12 +66,25 @@ Submit a Job without a name (automatically generates a name)
 runai submit -i gcr.io/run-ai-demo/quickstart -g 1 
 ```
 
+Submit a job without a name to an external URL:
```

Comment:
What is "a job without a name"?

Reply:
My assumption was that the user does not want to be "bothered" with making a unique name and relies on the autogenerated unique name. I can change this to:
```suggestion
Submit a job using the system autogenerated name to an external URL:
```